apisix/admin/plugins.lua

```diff
@@ -25,7 +25,7 @@ local type      = type
 local table_remove = table.remove
 local table_sort = table.sort
 local table_insert = table.insert
-
+local json_schema = {}
```

Comment:
`json_schema` should be a temporary variable in function.

Reply:
Yes, the previous wording is indeed wrong, it has been revised